src/Printable.java

```diff
@@ -0,0 +1,19 @@
+import java.lang.reflect.Field;
+
+public interface Printable {
+    default void print() {
+        Class<?> oClass = this.getClass();
+        Field[] oFields = oClass.getDeclaredFields();
+
+        System.out.println("Content of " + oClass.getSimpleName() + " is ");
+        for (Field f : oFields) {
+            f.setAccessible(true);
+            try {
+                Object fValue = f.get(this);
+                System.out.println(f.getName() + " - " + fValue);
+            } catch (IllegalAccessException e) {
+                e.printStackTrace();
+            }
+        }
+    }
```

Comment:
what is the main idea of this interface? 
Why such complex logic to just print a value? It's more suited for annotations.....or logic with reflection

Reply:
and where you use it?

Reply:
The main idea is to give any implementing class a method .print() its content to the console.Unfortunately, I don't have another idea for printing the content of any class in a different way. Instead of using  interface Printable, we could override toString() in every class, and then we wouldn't need the Printable interface.

Reply:
I don't use it now, but I can use it in AbstractUser for example

Reply:
I don't use it now, but I can use it in AbstractUser for example